rescript.json

```diff
@@ -1,23 +1,24 @@
 {
   "name": "coronate",
-  "reason": { "react-jsx": 3 },
+  "jsx": { "version": 4 },
   "sources": [
     { "dir": "src", "subdirs": true },
     { "dir": "__tests__", "type": "dev" },
     { "dir": "testutils", "type": "dev" }
   ],
-  "package-specs": [{ "module": "es6", "in-source": true }],
-  "suffix": ".bs.js",
+  "package-specs": [{ "module": "esmodule", "in-source": true }],
+  "suffix": ".res.js",
   "namespace": true,
   "bs-dependencies": [
+    "@rescript/core",
     "@ryyppy/rescript-promise",
     "@rescript/react",
-    "re-classnames",
     "rescript-webapi",
     "rescript-blossom"
   ],
+  "bsc-flags": ["-open RescriptCore"],
   "warnings": {
-    "number": "+A-48-42",
+    "number": "+A-48-42-44",
```

Comment:
The `open Belt` generates a ton of warnings otherwise, but I'm not the biggest fan of suppressing it either

Reply:
For this warning, you can suppress it case-by-case by using `open!` which is probably preferable to disabling it completely. https://rescript-lang.org/docs/manual/v11.0.0/module#use-open-to-ignore-shadow-warnings

Reply:
I just pushed a commit to address this.